### ISSUE TITLE
Website is not responsive for many pixels

### ISSUE THREAD

**Issue by techmannih** [mermaid-js/mermaid#4898]
### Description

hello @sidharthv96 @Yokozuna59,
I found an issue  in your website which is not responsive for 768px to 906px , navbar in responsible for this responsiveness.
Please you can check it ...I want to work on this issue .....please assign me

### Steps to reproduce

1.create small screens navbar for pixels from 768px to 906px
2.Navbar is not responsible for 768px to 906px

### Screenshots

![2023-10-03_19-49](https://github.com/mermaid-js/mermaid/assets/125847751/ae8932b8-a217-4d0c-b3ad-5dd940d61c42)
![2023-10-03_19-49_1](https://github.com/mermaid-js/mermaid/assets/125847751/dfff0071-1ad0-4da5-807d-9d4b706c5f30)

### Setup

- Mermaid version:
- Browser and Version: [Chrome, Edge, Firefox]

**Comment by Akshaybagai52** [comment]
Hello can this issue assign to me?

**Comment by Yokozuna59** [comment]
Hi there, @techmannih, Thanks for reporting the issue.
Feel free to create a PR to solve this issue. And please read our [contributing guidelines](http://mermaid.js.org/community/documentation.html) for docs first.

@Akshaybagai52 @techmannih As for assigning, please see https://github.com/mermaid-js/mermaid/issues/4487#issuecomment-1681607124.

**Comment by Yokozuna59** [comment]
> @Yokozuna59 Thanks for assigning will update you when it will be done

I don't how is that helpful. The issue would be already linked the PR.

> can you confirm me that will it count for hacktoberfest ?

Sorry, I don't know what is that, so I'm sure about that.

**Comment by Akshaybagai52** [comment]
> > @Yokozuna59 Thanks for assigning will update you when it will be done
> 
> I don't how is that helpful. The issue would be already linked the PR.
> 
> > can you confirm me that will it count for hacktoberfest ?
> 
> Sorry, I don't know what is that, so I'm sure about that.

Okay thanks

**Comment by techmannih** [comment]
Sure @Yokozuna59 I will create PR

**Comment by sidharthv96** [comment]
@Akshaybagai52, @techmannih was the one who created the issue, with the intention of working on it mentioned in the comment. 

Although we don't have explicit rules, once someone has expressed interest in fixing an issue, it would be the ideal thing to let them work on it first. 

Open source is built upon the ideas of collaboration and being considerate is a very important quality. :)

You can search for issues with the "Good First Issue" label to find more issues you can work with. There are ~800 other issues :)

**Comment by techmannih** [comment]
> @Akshaybagai52, @techmannih was the one who created the issue, with the intention of working on it mentioned in the comment.
> 
> Although we don't have explicit rules, once someone has expressed interest in fixing an issue, it would be the ideal thing to let them work on it first.
> 
> Open source is built upon the ideas of collaboration and being considerate is a very important quality. :)
> 
> You can search for issues with the "Good First Issue" label to find more issues you can work with. There are ~800 other issues :)

Yes @sidharthv96 , I was create this issue so I m working on it and I will create a PR soon
